modules/deriving/src/test/scala/shapeless3/deriving/deriving.scala

```diff
@@ -393,3 +393,52 @@ class DerivationTests:
     assertEquals(Right(ISB(42, "foo", true)), parser.parseShort("s=foo,i=42,b=true,hidden=?"))
     assertEquals(Left("Missing field 's';"), parser.parseShort("i=42,b=kinda"))
     assertEquals(Left("Invalid field 'broken';"), parser.parseShort("i=42,broken,?"))
+
+  @Test
+  def userDefinedMirrors(): Unit =
+    def assertImportSuggested(errors: List[testing.Error]): Unit =
+      assertEquals(1, errors.size)
+      val message = errors.head.message.trim
+      assert(message.startsWith("No given instance of type"))
+      assert(message.endsWith("Generic.fromMirror"))
```

Comment:
@tschuchortdev the default message already suggests the import so I think it's adequate

Reply:
Alright. In my experience those suggestions are rarely productive, so I tend to ignore them.